manager/cluster_config_env.py

```diff
@@ -15,13 +15,16 @@
 import sys
 import yaml
 import json
+from copy import deepcopy
 
 
 def export(base_key, value):
-    if base_key.lower().startswith("cortex_tags"):
-        inlined_tags = ",".join([f"{k}={v}" for k, v in value.items()])
-        print(f"export CORTEX_TAGS={inlined_tags}")
-        print(f"export CORTEX_TAGS_JSON='{json.dumps(value)}'")
+    if base_key.lower() == "cortex_tags":
+        exportTags(value, "CORTEX_TAGS")
+        exportTags(
+            value, "CORTEX_OPERATOR_LOAD_BALANCER_TAGS", {"cortex.dev/load-balancer": "operator"}
+        )
+        exportTags(value, "CORTEX_APIS_LOAD_BALANCER_TAGS", {"cortex.dev/load-balancer": "apis"})
```

Comment:
For consistency I think `CORTEX_APIS_LOAD_BALANCER_TAGS` should be renamed to `CORTEX_API_LOAD_BALANCER_TAGS`. Similarly for the value as well.

Reply:
some of the internals use "apis" (e.g. `ingressgateway-apis` and `apis-istio-gateway`), but I agree that "api" makes more sense for the user